### DISCUSSION TITLE
Property values do not have a tooltip when their length is out of the screen.

### DISCUSSION THREAD

**Discussion by CodingOctocat** [godotengine/godot-proposals#7680] [Editor]
As you can see in the picture, I can't see the full content, and no ToolTip shows the full content when I move the mouse over the value.

![image](https://github.com/godotengine/godot-proposals/assets/7220248/bd34ff51-ebc6-4d97-bc87-75384f716e7f)

**Comment by Zireael07** [comment]
Looks like a bug, not a feature request.

**Reply by Calinou** [reply]
I'd say it's technically a missing feature.
